Makefile

```diff
@@ -60,7 +60,7 @@ else
 endif
 # sed on Mac does not support the same syntax for in-place updates as sed on Linux
 # When running on MacOS it's best to install gsed and run Makefile with SED=gsed
-ifeq ($(GOOS),darwin)
+ifeq ($(shell uname -s),Darwin)
```

Comment:
we want the actual OS here, not what GOOS may have been overwritten with (since we call make recursively and override GOOS for cross-compiling)